File: app/src/main/java/b2w/com/alodjinha/Model/Banner.java
package b2w.com.alodjinha.Model;

public class Banner {

    private String id;
    private String urlImage;
    private String linkUrl;

    public Banner() {
    }

    public Banner(String id, String urlImage, String linkUrl) {
        this.id = id;
        this.urlImage = urlImage;
        this.linkUrl = linkUrl;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getUrlImage() {
        return urlImage;
    }

    public void setUrlImage(String urlImage) {
        this.urlImage = urlImage;
    }

    public String getLinkUrl() {
        return linkUrl;
    }

    public void setLinkUrl(String linkUrl) {
        this.linkUrl = linkUrl;
    }
}

File: app/src/main/java/br/com/andrecouto/alodjinha/domain/model/lodjinha/Banner.kt
package br.com.andrecouto.alodjinha.domain.model.lodjinha

import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey
import com.google.gson.annotations.SerializedName
import java.io.Serializable

@Entity(tableName = "banners")
data class Banner(@SerializedName("id") @PrimaryKey var id : Int,
                  @SerializedName("urlImagem") var urlImage : String?,
                  @SerializedName("linkUrl") var urlLink : String?) : Serializable
